### DISCUSSION TITLE
French translation for 3.1.9

### DISCUSSION THREAD

**Discussion by patrickus33** [templaza/astroid-framework#932] [General]
@sonvnn  @templaza/astroid 
Hi,
Here is the french translation for 3.1.9
Added lhe lines about ReCaptcha and turnstile and all others !
[fr-FR.astroid.ini.zip](https://github.com/user-attachments/files/18193827/fr-FR.astroid.ini.zip)

Merry Christmas to all of you !
Patrick

**Comment by sonvnn** [comment]
Thank you. I updated the package. Merry Christmas & Happy New Year!
